app/views/markets/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="side">
+  <h3>Vendors at <%= @market.name%>:</h3>
+  <table>
+    <% @vendors.each do |v| %>
+      <tr>
+        <% if @im_a_market %>
+          <td>
+            <%= link_to "Edit", edit_market_vendor_path(v.market_id, v.id), method: :get %>
+          </td>
+          <% end %>
+          <% if @im_a_market %>
+          <td>
+            <%= link_to "Delete",   "/markets/#{v.market_id}/vendors/#{v.id}", method: "delete",  data: { confirm: "Are you sure you want to delete this vendor?" } %>
```

Comment:
Instead of providing the path for this link explicitly, we should use one of the route helper methods.